### ISSUE TITLE
[SF_ACC][FileUploader]: Input is skipped in the reverse tabbing order in the dialog

### ISSUE THREAD

**Issue by ksblklu02** [SAP/ui5-webcomponents#9090]
### Bug Description

With JAWS, When user navigate with Tab key on all the input field until reaching Cancel button -> After press Shift+Tab and continue tabbing until reaching the FileUploader Input Upon pressing the Shift+Tab key again -> Observe that the focus skips the input field, resulting in an incorrect tab order.

### Affected Component

ui5-file-uploader

### Expected Behaviour

The tabbing order defined on the page is logical and tab order should not get trapped, Also user should be able to tab through the entire page and shift+tab back in the same reverse order.

### Isolated Example

https://stackblitz.com/edit/github-utmbsj?file=src%2FApp.tsx

### Steps to Reproduce

1. Go to https://stackblitz.com/edit/github-utmbsj?file=src%2FApp.tsx
2. Click"open dialog" button
3. Tabbing to the cancel button, and then use Shift + Tab reverse tab to the FileUploader component
4. See the FileUploader input was skipped in reverse tabbing order

### Priority

Medium

### UI5 Web Components Version

1.24.0

### Browser

Chrome

### Operating System

Mac OS

### Organization

SAP SuccessFactors

### Declaration

- [X] I’m not disclosing any internal or sensitive information.

**Comment by niyap** [comment]
Hello @SAP/ui5-webcomponents-topic-b,

When ui5-file-uploader without button but just an input field is placed within a dialog as a first element, it receives focus which I think is not expected, as it does not receive focus in any other use case. On backward navigation the same component is not focused.
Here is a sample: [Playground](https://sap.github.io/ui5-webcomponents/play/#eyJpbmRleC5odG1sIjp7Im5hbWUiOiJpbmRleC5odG1sIiwiY29udGVudCI6IjwhLS0gcGxheWdyb3VuZC1mb2xkIC0tPlxuPCFET0NUWVBFIGh0bWw-XG48aHRtbCBsYW5nPVwiZW5cIj5cblxuXG5cbjxoZWFkPlxuICAgIFxuICAgIDxzdHlsZT5cbiAgICAgICo6bm90KDpkZWZpbmVkKSB7XG4gICAgICAgIGRpc3BsYXk6IG5vbmU7XG4gICAgICB9XG4gICAgPC9zdHlsZT5cblxuICAgIFxuICAgIDxzdHlsZT5cbiAgICAgICo6bm90KDpkZWZpbmVkKSB7XG4gICAgICAgIGRpc3BsYXk6IG5vbmU7XG4gICAgICB9XG4gICAgPC9zdHlsZT5cblxuICAgIDxtZXRhIGNoYXJzZXQ9XCJVVEYtOFwiPlxuICAgIDxtZXRhIG5hbWU9XCJ2aWV3cG9ydFwiIGNvbnRlbnQ9XCJ3aWR0aD1kZXZpY2Utd2lkdGgsIGluaXRpYWwtc2NhbGU9MS4wXCI-XG4gICAgPHRpdGxlPlNhbXBsZTwvdGl0bGU-XG4gICAgPGxpbmsgcmVsPVwic3R5bGVzaGVldFwiIGhyZWY9XCIuL21haW4uY3NzXCI-XG48L2hlYWQ-XG5cbjxib2R5IHN0eWxlPVwiYmFja2dyb3VuZC1jb2xvcjogdmFyKC0tc2FwQmFja2dyb3VuZENvbG9yKTsgaGVpZ2h0OiA0NTBweDtcIj5cbiAgICA8IS0tIHBsYXlncm91bmQtZm9sZC1lbmQgLS0-XG5cbiAgICA8dWk1LWJ1dHRvbiBpZD1cImRpYWxvZ09wZW5lclwiPk9wZW4gRGlhbG9nPC91aTUtYnV0dG9uPlxuXG4gICAgPHVpNS1kaWFsb2cgaWQ9XCJkaWFsb2dcIiBoZWFkZXItdGV4dD1cIlJlZ2lzdGVyIEZvcm1cIj5cbiAgICAgICAgPHNlY3Rpb24gY2xhc3M9XCJsb2dpbi1mb3JtXCI-XG4gICAgICAgICAgICA8ZGl2PlxuICAgICAgICAgICAgICAgIDx1aTUtbGFiZWwgZm9yPVwidXNlcm5hbWVcIiByZXF1aXJlZCBzaG93LWNvbG9uPlVzZXJuYW1lPC91aTUtbGFiZWw-XG4gICAgICAgICAgICAgICAgPHVpNS1maWxlLXVwbG9hZGVyIGlkPVwidXNlcm5hbWVcIj48L3VpNS1maWxlLXVwbG9hZGVyPlxuICAgICAgICAgICAgPC9kaXY-XG4gICAgICAgICAgICA8ZGl2PlxuICAgICAgICAgICAgICAgIDx1aTUtbGFiZWwgZm9yPVwicGFzc3dvcmRcIiByZXF1aXJlZCBzaG93LWNvbG9uPlBhc3N3b3JkPC91aTUtbGFiZWw-XG4gICAgICAgICAgICAgICAgPHVpNS1pbnB1dCBpZD1cInBhc3N3b3JkXCIgdHlwZT1cIlBhc3N3b3JkXCIgdmFsdWUtc3RhdGU9XCJFcnJvclwiPjwvdWk1LWlucHV0PlxuICAgICAgICAgICAgPC9kaXY-XG4gICAgICAgICAgICA8ZGl2PlxuICAgICAgICAgICAgICAgIDx1aTUtbGFiZWwgZm9yPVwiZW1haWxcIiB0eXBlPVwiRW1haWxcIiByZXF1aXJlZCBzaG93LWNvbG9uPkVtYWlsPC91aTUtbGFiZWw-XG4gICAgICAgICAgICAgICAgPHVpNS1pbnB1dCBpZD1cImVtYWlsXCI-PC91aTUtaW5wdXQ-XG4gICAgICAgICAgICA8L2Rpdj5cbiAgICAgICAgICAgIDxkaXY-XG4gICAgICAgICAgICAgICAgPHVpNS1sYWJlbCBmb3I9XCJhZGRyZXNzXCIgc2hvdy1jb2xvbj5BZGRyZXNzPC91aTUtbGFiZWw-XG4gICAgICAgICAgICAgICAgPHVpNS1pbnB1dCBpZD1cImFkZHJlc3NcIj48L3VpNS1pbnB1dD5cbiAgICAgICAgICAgIDwvZGl2PlxuICAgICAgICA8L3NlY3Rpb24-XG4gICAgICAgIDxkaXYgc2xvdD1cImZvb3RlclwiIHN0eWxlPVwiZGlzcGxheTogZmxleDsganVzdGlmeS1jb250ZW50OiBmbGV4LWVuZDsgd2lkdGg6IDEwMCU7IGFsaWduLWl0ZW1zOiBjZW50ZXJcIj5cbiAgICAgICAgICAgIDxkaXYgc3R5bGU9XCJmbGV4OiAxO1wiPjwvZGl2PlxuICAgICAgICAgICAgPHVpNS1idXR0b24gY2xhc3M9XCJkaWFsb2dDbG9zZXJcIiBkZXNpZ249XCJFbXBoYXNpemVkXCI-UmVnaXN0ZXI8L3VpNS1idXR0b24-XG4gICAgICAgIDwvZGl2PlxuICAgIDwvdWk1LWRpYWxvZz5cbiAgICA8IS0tIHBsYXlncm91bmQtZm9sZCAtLT5cbiAgICA8c2NyaXB0IHR5cGU9XCJtb2R1bGVcIiBzcmM9XCJtYWluLmpzXCI-PC9zY3JpcHQ-XG48L2JvZHk-XG5cbjwvaHRtbD5cbjwhLS0gcGxheWdyb3VuZC1mb2xkLWVuZCAtLT5cbiJ9LCJtYWluLmpzIjp7Im5hbWUiOiJtYWluLmpzIiwiY29udGVudCI6Ii8qIHBsYXlncm91bmQtaGlkZSAqL1xuaW1wb3J0IFwiLi9wbGF5Z3JvdW5kLXN1cHBvcnQuanNcIjtcbi8qIHBsYXlncm91bmQtaGlkZS1lbmQgKi9cbmltcG9ydCBcIkB1aTUvd2ViY29tcG9uZW50cy9kaXN0L0RpYWxvZy5qc1wiO1xuaW1wb3J0IFwiQHVpNS93ZWJjb21wb25lbnRzL2Rpc3QvQnV0dG9uLmpzXCI7XG5pbXBvcnQgXCJAdWk1L3dlYmNvbXBvbmVudHMvZGlzdC9JbnB1dC5qc1wiO1xuaW1wb3J0IFwiQHVpNS93ZWJjb21wb25lbnRzL2Rpc3QvTGluay5qc1wiO1xuaW1wb3J0IFwiQHVpNS93ZWJjb21wb25lbnRzL2Rpc3QvTGFiZWwuanNcIjtcbmltcG9ydCBcIkB1aTUvd2ViY29tcG9uZW50cy9kaXN0L0ZpbGVVcGxvYWRlci5qc1wiO1xuXG5cbnZhciBkaWFsb2dPcGVuZXIgPSBkb2N1bWVudC5nZXRFbGVtZW50QnlJZChcImRpYWxvZ09wZW5lclwiKTtcbnZhciBkaWFsb2cgPSBkb2N1bWVudC5nZXRFbGVtZW50QnlJZChcImRpYWxvZ1wiKTtcbnZhciBkaWFsb2dDbG9zZXJzID0gWy4uLmRpYWxvZy5xdWVyeVNlbGVjdG9yQWxsKFwiLmRpYWxvZ0Nsb3NlclwiKV07XG5cbmRpYWxvZ09wZW5lci5hY2Nlc3NpYmlsaXR5QXR0cmlidXRlcyA9IHtcbiAgICBoYXNQb3B1cDogXCJkaWFsb2dcIixcbiAgICBjb250cm9sczogZGlhbG9nLmlkLFxufTtcbmRpYWxvZ09wZW5lci5hZGRFdmVudExpc3RlbmVyKFwiY2xpY2tcIiwgKCkgPT4ge1xuICAgIGRpYWxvZy5vcGVuID0gdHJ1ZTtcbn0pO1xuZGlhbG9nQ2xvc2Vycy5mb3JFYWNoKGJ0biA9PiB7XG4gICAgYnRuLmFkZEV2ZW50TGlzdGVuZXIoXCJjbGlja1wiLCAoKSA9PiB7XG4gICAgICAgIGRpYWxvZy5vcGVuID0gZmFsc2U7XG4gICAgfSk7XG59KSJ9LCJtYWluLmNzcyI6eyJuYW1lIjoibWFpbi5jc3MiLCJjb250ZW50IjoiLmxvZ2luLWZvcm0ge1xuICAgIGRpc3BsYXk6IGZsZXg7XG4gICAgZmxleC1kaXJlY3Rpb246IGNvbHVtbjtcbiAgICBqdXN0aWZ5LWNvbnRlbnQ6IHNwYWNlLWV2ZW5seTtcbiAgICBhbGlnbi1pdGVtczogZmxleC1zdGFydDtcbiAgICBtYXJnaW46IDNyZW0gNnJlbTtcbn1cblxuLmxvZ2luLWZvcm0gPiBkaXYge1xuICAgIGRpc3BsYXk6IGdyaWQ7XG4gICAgd2lkdGg6IDE1cmVtO1xuICAgIG1hcmdpbi1ib3R0b206IC41cmVtO1xufVxuXG5AbWVkaWEobWF4LXdpZHRoOiA2MDBweCkge1xuICAgIC5sb2dpbi1mb3JtIHtcbiAgICAgICAgbWFyZ2luOiAzcmVtIDFyZW07XG4gICAgfVxufSJ9fQ)

The issue can be reproduced in the provided StackBlitz sample as well.

Could you please check?

Thank you in advance!

Kind Regards,
Niya
